src/locales/en-US/stream.ftl

```diff
@@ -803,3 +795,37 @@ profile-changeEmail-saveChanges = Save changes
 profile-changeEmail-email = Email
 profile-changeEmail-title = Email address
 profile-changeEmail-success = Your email has been successfully updated
+
+## Ratings and Reviews
+
+rr-reviewsTab = Reviews
```

Comment:
I don't love `rr` as a prefix here. We've generally been fairly explicit in our localization keys.

Reply:
Fair! I'll see what I can do!